### ISSUE TITLE
Folder icons blur inferior to GNOME 42's built-in blur (v30)

### ISSUE THREAD

**Issue by retrixe** [aunetx/blur-my-shell#211]
Old one, part of GNOME 42:
![Screenshot from 2022-04-08 17-15-02](https://user-images.githubusercontent.com/6884828/162430807-5498d786-e9f6-40fd-b57b-48bdca73a234.png)

New one provided by Blur My Shell v30:
![Screenshot from 2022-04-08 17-21-18](https://user-images.githubusercontent.com/6884828/162430888-3cfaa482-7fd9-4d51-b33b-92ae0a177c0b.png)

Disabling component-specific blurs is not a good workaround, since I would prefer the dash and others to remain blurred. Some sort of toggle so the old behaviour could be restored would be nice.

**Comment by aunetx** [comment]
Thanks for testing, I know this can look worse than the previous prefs with some wallpapers... I will need to test some more, and I may provide a settings to choose between darker and lighter backgrounds in order to accommodate for this, but this is really hard to choose somethings that feels good for everyone...

**Comment by aunetx** [comment]
Is this better with last commit? If yes, it will probably be set as default -- but later I will provide in the preferences an option to choose between different styles, as it is pretty important for everybody :)

**Comment by retrixe** [comment]
The new style is much better for folder icons, although I think the old dash background from older versions of the extension was better than the new one.

Old (v30 and older):
![Screenshot from 2022-04-11 21-56-42](https://user-images.githubusercontent.com/6884828/162786734-068a7a4d-928f-4ff4-9d38-62e4b012e1b5.png)

New (v31):
![Screenshot from 2022-04-11 21-56-55](https://user-images.githubusercontent.com/6884828/162786785-e62de9b9-a506-4b4d-9a54-c81d6dced98a.png)

On darker backgrounds:

Old (v30 and older):
![image](https://user-images.githubusercontent.com/6884828/162788322-631a6686-ea9e-4edb-819e-7fe08d1ef778.png)

New (v31):
![image](https://user-images.githubusercontent.com/6884828/162788420-8c549466-9339-446a-94ac-b33198175c7a.png)

**Comment by aunetx** [comment]
You're right, but tbh I would really prefer not to have different styles for the dash, the app folders and the search entry, so I think the current solution is good enough... And I just added a switch to select whether to use a dark or light style, if you want to try if yo can install from the last commit!
